xml/System.Threading.Tasks/Task.xml

```diff
@@ -2095,7 +2095,7 @@ End Sub
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If the cancellation token is signaled before the specified time delay, a <xref:System.Threading.Tasks.TaskCanceledException> exception results, and the task is completed in the <xref:System.Threading.Tasks.TaskStatus?displayProperty=fullName> state.  Otherwise, the task is completed in the <xref:System.Threading.Tasks.TaskStatus?displayProperty=fullName> state once the specified time delay has elapsed.  
+ If the cancellation token is signaled before the specified time delay, a <xref:System.Threading.Tasks.TaskCanceledException> exception results, and the task is completed in the <xref:System.Threading.Tasks.TaskStatus.Canceled> state.  Otherwise, the task is completed in the <xref:System.Threading.Tasks.TaskStatus.RanToCompletion> state once the specified time delay has elapsed.  
```

Comment:
The two xrefs should be `<xref:System.Threading.Tasks.TaskStatus.Canceled?displayProperty=fullName>` and `<xref:System.Threading.Tasks.TaskStatus.RanToCompletion?displayProperty=fullName>`.